unrpyc.py

```diff
@@ -187,64 +188,133 @@ def worker(args, filename):
             print(traceback.format_exc())
         return False
 
-def check_inpath(inp, strict=True):
-    """Helper to check if given path exists and cast it to pathlikes."""
-    return pt(inp).resolve(strict)
-
-def main():
-    # python27 unrpyc.py [-c] [-d] [--python-screens|--ast-screens|--no-screens] file [file ...]
-    cc_num = cpu_count()
-    parser = argparse.ArgumentParser(description="Decompile .rpyc/.rpymc files")
-
-    parser.add_argument('-c', '--clobber', dest='clobber', action='store_true',
-                        help="overwrites existing output files")
-
-    parser.add_argument('-d', '--dump', dest='dump', action='store_true',
-                        help="instead of decompiling, pretty print the ast to a file")
-
-    parser.add_argument('-p', '--processes', dest='processes', action='store', type=int,
-                        choices=list(range(1, cc_num)), default=cc_num - 1 if cc_num > 2 else 1,
-                        help="use the specified number or processes to decompile."
-                        "Defaults to the amount of hw threads available minus one, disabled when muliprocessing is unavailable.")
-
-    parser.add_argument('-t', '--translation-file', dest='translation_file', action='store', default=None,
-                        help="use the specified file to translate during decompilation")
 
-    parser.add_argument('-T', '--write-translation-file', dest='write_translation_file', action='store', default=None,
-                        help="store translations in the specified file instead of decompiling")
+def set_cpu_num():
+    """Sets the number of used CPUs.
+    3/4 of available, 1 if cores below 3. Overall max is 8."""
```

Comment:
I'm not sure why we're changing this. (and slowing down running my testcases for no reason). If your scheduler cannot handle multiple active threads that's your OS's problem, not ours.